tests/services/test_users.py

```diff
@@ -61,12 +64,20 @@ def put_api_mock_response(self, mocker):
         return Py42Response(response)
 
     @pytest.fixture
-    def post_api_mock_error_response(self, mocker):
-        response = mocker.MagicMock(spec=Response)
-        response.status_code = 500
-        response.encoding = "utf-8"
-        response.text = MOCK_ERROR_text
-        return Py42Response(response)
+    def internal_server_error(self, mocker):
+        base_error = mocker.MagicMock(spec=HTTPError)
+        base_error.response = mocker.MagicMock(spec=Response)
+        return Py42InternalServerError(base_error)
```

Comment:
The error was never getting raised in the methods, and the tests were not really correctly checking so anyway (false positives)